tools/sortorbitals.py.in

```diff
@@ -0,0 +1,84 @@
+#!@PYTHON_EXECUTABLE@
+import re, subprocess, os 
+
+def orbsorter(blockbuff):
+
+    orbletter=blockbuff[0].split()[1]
+
+    if 's' in orbletter:
+        return 0
+    if 'p' in orbletter:
+        return 1
+    if 'd' in orbletter:
+        return 2
+    if 'f' in orbletter:
+        return 3
+    if 'g' in orbletter:
+        return 4
+    if 'h' in orbletter:
+        return 5
+    if 'i' in orbletter:
+        return 6
+    else:
+        print("There seems to be something wrong with the basis file...")
+        return 100
```

Comment:
I would do this with a dictionary: `{"s": 0, "p": 1, ...}` and:
```python
if orbletter in l_value:
    return l_value[orbletter]
else:
    # throw error
```

Also I would prefer the code to stop rather than to return L value of 100 and continue.